rules/windows/process_creation/proc_creation_win_lolbin_customshellhost.yml

```diff
@@ -0,0 +1,27 @@
+title: Suspicious CustomShellHost Execution
+id: 84b14121-9d14-416e-800b-f3b829c5a14d
+status: experimental
+description: Detects the execution of CustomShellHost binary where the child isn't located in 'C:\Windows\explorer.exe'
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/pull/180
+author: Nasreddine Bencherchali
+date: 2022/08/19
+tags:
+    - attack.defense_evasion
+    - attack.t1216
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|endswith: '\CustomShellHost.exe'
+        - OriginalFileName: 'CustomShellHost.exe'
+    filter:
+        - Image: 
+            - 'C:\Windows\explorer.exe'
+            - 'C:\Windows\System32\explorer.exe'
+        - CurrentDirectory|startswith: C:\Windows\System32\
```

Comment:
@frack113 : like this?

Reply:
Image can not be CustomShellHost.exe  and explorer.exe in the same time
it's two different events.

CustomShellHost.exe  load explorer from the CurrentDirectory.

the easiest way I think is:
```yaml
    selection:
        ParentImage|endswith: '\CustomShellHost.exe'
    filter:
        Image: 'C:\Windows\explorer.exe'
    condition: selection and not filte
```

I am not sure with CurrentDirectory there will not be many FP

Reply:
It loads explorer from default dir correct that's but if no explorer is found it loads from C:\Windows if we filter this we can find other calls (which shouldn't be that common) and if we find them we could filter them out.